packages/flutter/lib/src/painting/text_style.dart

```diff
@@ -562,6 +563,9 @@ class TextStyle extends Diagnosticable {
   /// equivalent as order produces differing transparency.
   final List<ui.Shadow> shadows;
 
+  /// A list of [FontFeature]s that affect how the font selects glyphs.
```

Comment:
There should be an explanation of how to use this here and in the TextStyle class docs. It is not immediately obvious from this short description

Reply:
Added some more info.  This property's docs will link to the `ui.FontFeature` docs, which offer a more complete explanation:
    https://github.com/flutter/engine/blob/master/lib/ui/text.dart#L106

Reply:
I guess that is ok since this feature is only likely to be used by more advanced users. Fully exposing docs here may add clutter and make it harder to find the more commonly used properties.